source/voice_control/custom_sentences.markdown

```diff
@@ -15,21 +15,13 @@ related:
 
 You may add your own sentences to the intent recognizer by either extending an [existing intent](https://developers.home-assistant.io/docs/intent_builtin/) or creating a new one. You may also [customize responses](#customizing-responses) for existing intents.
 
-## Adding a custom sentence to trigger an automation
+## Prerequisites
```

Comment:
```suggestion
## Adding a custom sentence to trigger an automation
```
I don't think this should be replaced with the prereqs here. It is the task title, indicating what it is exactly we are going to do here. Prerequisites should not standalone but be part of a task.

Alternatively, to keep the prereqs at this level, the top-level title could be changed to "Adding custom sentences to trigger an automation". Then it's clear what the prereqs are for.

Reply:
Understood, and my main goal is to keep the prerequs at this level, since its important that its considered before starting to make changes. But your suggestion is perfect.